app/mobile/src/App.tsx

```diff
@@ -0,0 +1,54 @@
+import { StatusBar } from "expo-status-bar";
+import React, { useState } from "react";
+import { StyleSheet, Text, View, Image, TouchableOpacity } from "react-native";
+import LandingScreen from "./components/LandingScreen";
+import StorybookUIRoot from "./storybook";
+
+export default function App() {
+  const [isOpen, setIsOpen] = useState(false);
+
+  return (
+    <View style={styles.container}>
+      <TouchableOpacity
+        onPress={() => setIsOpen(!isOpen)}
+        style={{
+          position: "absolute",
+          top: 30,
+          right: 10,
+          backgroundColor: "red",
+          zIndex: 99,
+          borderRadius: 25,
+        }}
+      >
+        <Text
+          style={{
+            height: 20,
+            width: 20,
+            display: "flex",
+            justifyContent: "center",
+            textAlign: "center",
+          }}
+        >
+          {isOpen ? "X" : "SB"}
```

Comment:
Shouldn't we only show this in dev mode? Don't think we should expose Storybook publicly like this in the prod version of the app

Reply:
The current 'deploy on tag' thing is only for internal use and I would like to keep storybook there for now in case someone wants to use it. We have created ticket #2015 to address this issue so that we can have storybook on the preview 'release channel', and turn it off for the production release channel. We'll configure via an env variable or something.